toolbox.mk

```diff
@@ -172,11 +174,16 @@ terraform-destroy terraform-console: terraform-ensure-workspace
 ##
 ## Buildkite targets.
```

Comment:
Very minor but would you mind adding the `that DO require a workspace.` / `that DON'T require a workspace.` comment header for each of these Buildkite targets now that there's one that needs a WORKSPACE?